test/edu/cscc/CustomerRepositoryImplTest.java

```diff
@@ -0,0 +1,79 @@
+package edu.cscc;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CustomerRepositoryImplTest {
+
+    CustomerRepository customerRepository = new CustomerRepositoryImpl();
+
+    Customer customer1 = new Customer("Mary", "Smith", "marySmith2022@gmail.com");
+    Customer customer2 = new Customer("Joe", "Smith", "joeSmith2022@gmail.com");
+    Customer customer3 = new Customer("Jane", "Smith", "joeSmith2022@gmail.com");
+
+    @Test
+    void itCanCreateCustomerEntry() throws DuplicateCustomerException {
+        customer1 = customerRepository.create(customer1);
```

Comment:
This test needs assert statements to check that the customer was actually created. What you could do is to call `customerRepository.read(customer1.getAccountNumber());`, and the assert that the record matches the one that was just created.